chainerx_cc/chainerx/cuda/cudnn.h

```diff
@@ -10,21 +12,40 @@
 #include "chainerx/macro.h"
 #include "chainerx/stack_vector.h"
 
+#define CHAINERX_CUDA_CUDNN_CALL_WITH_HANDLE(handle, func, args...)                                                                    \
```

Comment:
Actually, I don't think we need this macro? We can simply use `CHAINERX_CUDA_CUDNN_CALL` passing the handle similar to any other argument to the cuDNN routine. It's easier from the caller perspective I think.

Reply:
I don't think `CudnnHandle::Call` can be bypassed.
It has a concurrency control.

Reply:
Ah, that is true of course. Thanks.